indexing-hadoop/src/main/java/io/druid/indexer/hadoop/SegmentLoadSpec.java

```diff
@@ -0,0 +1,111 @@
+package io.druid.indexer.hadoop;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.api.client.repackaged.com.google.common.base.Preconditions;
+import io.druid.granularity.QueryGranularity;
+import io.druid.query.filter.DimFilter;
+import io.druid.segment.column.ValueType;
+import org.joda.time.Interval;
+
+import java.util.List;
+
+public class SegmentLoadSpec
```

Comment:
this is different from DataSegment, this is the specification of what druid data should be read and not the details of particular segment. I should probably rename this class to DatasourceIngestionSpec.java.